models/eprop_iaf_adapt.h

```diff
@@ -92,58 +97,69 @@ able to spike for an absolute refractory period :math:`t_\text{ref}`.
 An additional state variable and the corresponding differential equation
 represents a piecewise constant external current.
 
-Furthermore, the pseudo-derivative of the membrane voltage needed for e-prop
-plasticity is calculated:
-
-.. math::
-    \psi_j^t = \frac{\gamma}{v_\text{th}} \text{max}
-               \left(0, 1-\left| \frac{v_j^t-A_j^t}{v_\text{th}}\right| \right) \,.
-
-See the documentation on the ``iaf_psc_delta`` neuron model for more information
-on the integration of the subthreshold dynamics.
+See the documentation on the :doc:`iaf_psc_delta<../models/iaf_psc_delta/>` neuron model
+for more information on the integration of the subthreshold dynamics.
 
-The change of the synaptic weight is calculated from the gradient :math:`g` of
-the loss :math:`E` with respect to the synaptic weight :math:`W_{ji}`:
-:math:`\frac{\mathrm{d}{E}}{\mathrm{d}{W_{ij}}}=g`
+The change of the synaptic weight is calculated from the gradient :math:`g^t` of
+the loss :math:`E^t` with respect to the synaptic weight :math:`W_{ji}`:
+:math:`\frac{\text{d}{E^t}}{\text{d}{W_{ij}}}=g^t`
 which depends on the presynaptic
 spikes :math:`z_i^{t-1}`, the surrogate gradient / pseudo-derivative of the postsynaptic membrane
 voltage :math:`\psi_j^t` (which together form the eligibility trace
 :math:`e_{ji}^t`), and the learning signal :math:`L_j^t` emitted by the readout
 neurons.
 
+See the documentation on the :doc:`eprop_archiving_node<../models/eprop_archiving_node/>` for details on the surrogate
+gradients functions.
+
+In between two presynaptic spikes, the gradient is calculated for each time step until the cutoff time point, i.e., for
+:math:`t \, \epsilon \, \left[ t_\text{s}^{t-1}, \text{min} \left( t_\text{s}^{t-1} + {\Delta t}_\text{c},
+t_\text{s}^t \right) \right]`:
```

Comment:
```suggestion
In the interval between two presynaptic spikes, the gradient is calculated
at each time step until the cutoff time point. This computation occurs over
the time range:
:math:`t \in \left[ t_\text{prev}, \min \left( t_\text{prev} + \Delta t_\text{c}, t_\text{curr} \right) \right]`.
Here, :math:`t_\text{prev}` represents the time of the previous spike that
stimulated the synapse, while :math:`t_\text{curr}` is the time of the
current spike, which triggers the application of the learning rule and the
subsequent synaptic weight update. The cutoff :math:`\Delta t_\text{c}` 
defines the maximum allowable interval for integration between spikes. 
The expression for the gradient is given by
```
The variables $t_\text{s}^{t-1}$ and $t_\text{s}^{t}$ might be confusing because they include a variable named $t$ with a superscript $t$. I suggest renaming them to $t_\text{prev}$ and $t_\text{curr}$ for better clarity. Although the meanings of these variables can be inferred from the context,  I think it would be more precise and formal to explicitly state what they represent. I think also that $\in$ should be preferable over $\epsilon$.

Reply:
Fixed in https://github.com/JesusEV/nest-simulator/pull/16/commits/06a855a09cbb1963f3f4c1a49ac30baaa5ec4b73